src/framework/uicomponents/qml/MuseScore/UiComponents/PageTabButton.qml

```diff
@@ -145,10 +150,12 @@ RadioDelegate {
             StyledTextLabel {
                 id: textLabel
                 anchors.verticalCenter: parent.verticalCenter
-
-                horizontalAlignment: Text.AlignLeft
+ 
+                horizontalAlignment: root.isVertical ? Text.AlignHCenter : Text.AlignLeft
                 font: root.normalStateFont
                 text: root.title
+                elide: Text.ElideNone
```

Comment:
One more question, why is this line necessary?

Reply:
Due to the existence of padding in the button itself, not having that line causes the text to be truncated when the button is selected.
The alternative was to add the padding to the calculation of the Text Component Width (as was originially done in when the bug existed), but I found that doing so caused the button to be unnecessarily large for the text within it. I found the implemented alternative more appealing.
As mentioned previously, I checked with the other uses of the pageTabButton and it appeared to make no difference, but I can alter the code to the Alternative if you are concerned with it provoking unwanted changes, specially in the MacOS environment, as I'm unable to test for it.

Reply:
Due to the way truncation is calculated, having the text box width be the exact width of the text (in other words, when the text has the Selected Font) cause the text to be truncated.

The two possible solutions that came about were to add 1 to the newly calculated width (which didn't seem like ideal practice in general) or to remove the truncation from the text in this situation (which, upon examining the remaining uses of the PageTabButton, did not create a problem, but I am unable to experiment on OS's other than Linux and Windows).

One additional option that I could make use of is remove the truncation only if the width of the text is exactly equal to the calculated width, but it would be an unnecessary check that didn't entirely make sense in theory.

If there is something I am missing (such as the text being truncated due to a pre-defined margin on text-boxes) please let me know, and I'll attempt to use that information to work on a solution that is equal to neither of the aforementioned

Reply:
I found that `advanceWidth` produces the expected result on macOS. I pushed that solution to this PR. Could you try if that works for you too?

Reply:
The text isn«t being truncated, but I also believe it isn't being boldened when selected. Not sure if it's from the change or a different commit made somewhere along the way by a different contributor

Reply:
> I also believe it isn't being boldened when selected

That seems to work fine on my side. I'll ask one of our testers to give this a final test before merging.